packages/alert/src/Alert.tsx

```diff
@@ -58,6 +59,13 @@ const iconFills = {
   informative: colors.informative50(),
 }
 
+const iconBase = {
```

Comment:
Nit: Can we rename this to `icons`?

Reply:
Nice! I didn't know that was possible to return React components this way. Done in f7cac55ff14ff68505d84df8d762cacb54f59b60